README.md

```diff
@@ -1,2 +1,145 @@
 # redux-needs &middot; [![Build Status](https://travis-ci.org/LUKKIEN/redux-needs.svg?branch=master)](https://travis-ci.org/LUKKIEN/redux-needs) [![codecov](https://codecov.io/gh/LUKKIEN/redux-needs/branch/master/graph/badge.svg)](https://codecov.io/gh/LUKKIEN/redux-needs)
 
+> 🎁 Wrapping everyday data to your containers in need 🎁
+
+Bind actions to changes in your redux state based on the needs of your active
+redux components.
+
+[Usage](#usage) |
+[API](#api) |
+[Installation](#installation) |
```

Comment:
Can we move installation to the top? Makes more sense in the flow: install, usage, api